vips/vip-350/vip-350/commands.ts

```diff
@@ -0,0 +1,44 @@
+import { Assets, BaseAssets } from "./addresses";
+
+type IncentiveAndAccessibility = [number, number];
+
+function getIncentiveAndAccessibility(baseAsset: string) {
+  const incentivesAndAccess: Array<[number, number]> = [];
+
+  for (const asset of Assets) {
+    if (asset != baseAsset) {
+      // Every conversion of the baseAsset for USDT in a SingleTokenConverter with a baseAsset != USDT is enabled only for converters,
+      // because the RiskFundConverter of the USDTPrimeConverter should be able to cover those conversions
```

Comment:
The distribution of funds to the `RiskFundConverter` is zero since [VIP-332](https://app.venus.io/#/governance/proposal/332?chainId=56), so I think we should not configure only private conversions when `asset` is USDT and `baseAsset` is not XVS (the `USDTPrimeConverter` is not going to buy enough USDT from the rest of the prime converters)

Reply:
Refactored code, set incentive and access - `[1e14, 1]` for all `baseAsset` to `USDT` pairs: 9a3e99117264322f8b68ae61dbe4ea45c7691e9d